### DISCUSSION TITLE
Does Auto support GitLab CI/CD?

### DISCUSSION THREAD

**Discussion by cakriwut** [intuit/auto#2335] [General]
I am looking into all documentation and sample, and it seems that auto only support github. Is there any chance that I can use it in my Gitlab CI/CD and publish npm package to gitlab package repository?

Any guidance will be appreciated.

**Comment by hipstersmoothie** [comment]
Not right now. Auto is heavily built around octokit. I know there is a similar package for gitlab but I myself don't use that platform and probably won't be the one to implement that. If someone did want to implement it though I would definitely help with guidance

**Reply by jdalrymple** [reply]
You can use gitbeaker under the hood! but it would require a wrapper to be a plugin
